terraform/environments/data-platform/iam.tf

```diff
@@ -41,8 +73,9 @@ data "aws_iam_policy_document" "athena_load_lambda_function_policy" {
       "s3:GetBucketLocation"
     ]
     resources = [
-      "${module.s3-bucket.bucket.arn}/*",
-      "${module.s3-bucket.bucket.arn}",
+      "${module.data_s3_bucket.bucket.arn}/raw/*",
```

Comment:
will need `"${module.data_s3_bucket.bucket.arn}/curated/*"` too

Reply:
Yes we do - will add this